### ISSUE TITLE
Problem with german output of git commands

### ISSUE THREAD

**Issue by gaudes** [AlCalzone/release-script#16]
Hi,

as written in Discord.

The output of my git command "git status" is in german:

```
Auf Branch master
Ihr Branch ist auf demselben Stand wie 'origin/master'.

nichts zu committen, Arbeitsverzeichnis unverändert
```

When running the release-script with the following command:

```
npm run release minor -- --dry --all
```

I receive the following message:

```
Your branch has uncommitted changes that will be included in the release commit!
```

Probably a problem with the german output of the git command ?

Thanks and regards,

Ralf

**Comment by AlCalzone** [comment]
Notes for me:
* Detect commit differences: `git rev-list --left-right --count HEAD...origin/master`
* machine readable check for changed files: `git status --porcelain`

**Comment by AlCalzone** [comment]
should be fixed in v1.7.1, please test
